assets/js/trivyScans.js

```diff
@@ -1,152 +1,141 @@
 jQuery(function () {
   const columns = [
     {
-      data: 'type',
+      data: 'attributes',
       createdCell: function (td, _cellData, rowData) {
-        const name = rowData?.attributes?.type
-        $(td).html(name)
+        const name = rowData?.attributes?.trivy_scan?.data?.attributes?.name || 'N/A'
+        $(td).html(`<a href="/trivy-scans/${name}">${name}</a>`)
       },
     },
     {
-      data: 'name',
+      data: 'attributes',
       createdCell: function (td, _cellData, rowData) {
-        const name = rowData?.attributes?.trivy_scan?.data?.attributes?.name || 'N/A'
-        $(td).html(`<a href="/trivy-scans/${name}">${name}</a>`)
+        const type = rowData?.attributes?.type || 'N/A'
+        $(td).html(type)
       },
     },
     {
-      data: 'build_image_tag',
+      data: 'attributes',
       createdCell: function (td, _cellData, rowData) {
         const buildImageTag = rowData?.attributes?.trivy_scan?.data?.attributes?.build_image_tag || 'N/A'
         $(td).html(buildImageTag)
       },
     },
     {
-      data: 'attributes.trivy_scan.data.attributes.trivy_scan_timestamp',
+      data: 'attributes',
       createdCell: function (td, _cellData, rowData) {
         const scan_timestamp = rowData?.attributes?.trivy_scan?.data?.attributes?.trivy_scan_timestamp || 'N/A'
         $(td).html(scan_timestamp)
       },
     },
     {
-      data: 'fixed_critical',
-      name: 'fixed_critical',
+      data: 'attributes',
       createdCell: function (td, _cellData, rowData) {
-        const summary = rowData?.attributes?.trivy_scan?.data?.attributes?.scan_summary?.scan_summary.summary || {}
+        const summary = rowData?.attributes?.trivy_scan?.data?.attributes?.scan_summary?.scan_summary?.summary || {}
         const os_fixed_critical = summary?.['os-pkgs']?.fixed?.CRITICAL ?? 0
         const lang_fixed_critical = summary?.['lang-pkgs']?.fixed?.CRITICAL ?? 0
         const value = os_fixed_critical + lang_fixed_critical
         $(td).html(`${value}`)
       },
     },
     {
-      data: 'fixed_high',
-      name: 'fixed_high',
```

Comment:
some columns have name as well but others don't - just removed as didn't seem to be used